logger/plotter.py

```diff
@@ -113,14 +120,14 @@ def _plot_xy(self, name, tag, x, y, time_idx=True):
             return True
         else:
             if self.unsafe_send:
-                args = {"Y": y, "X": x, "name": tag, "win": self.windows[name], "append": self.append}
+                args = {"Y": y, "X": x, "name": tag, "win": self.windows[name], "update": "append"}
                 self.worker_queue.put(args)
                 # Assume that the sending went right
                 return True
             else:
-                return bool(self.viz.updateTrace(Y=y, X=x, name=tag,
-                                                 win=self.windows[name],
-                                                 append=self.append))
+                return bool(self.viz.line(Y=y, X=x, name=tag,
```

Comment:
Is the `line` method backward compatible with older visdom versions?
Keep in mind that people using older python are stuck with visdom 0.1.7 because of Tornado not supporting old python encryption.

Reply:
Not sure. Will check when I get the time.

I think it should be alright, given that `updateTrace` was deprecated since last year according to https://github.com/facebookresearch/visdom/pull/342 while visdom 0.1.7 was released in January according to https://pypi.org/project/visdom/#history.